plugins/API/vue/src/Glossary/Glossary.vue

```diff
@@ -107,9 +107,11 @@ export default defineComponent({
   mounted() {
     const root = this.$refs.root as HTMLElement;
 
-    $('.scrollspy', root).scrollSpy();
-    $('.pushpin', root).pushpin({ top: $('.pushpin', root).offset()!.top });
-    $('.tabs', root).tabs();
+    setTimeout(() => {
+      $('.scrollspy', root).scrollSpy();
+      $('.pushpin', root).pushpin({ top: $('.pushpin', root).offset()!.top });
+      $('.tabs', root).tabs();
+    });
```

Comment:
Would this also work? Seems like a more Vue-way.
```suggestion
    this.$nextTick(() => {
      $('.scrollspy', root).scrollSpy();
      $('.pushpin', root).pushpin({ top: $('.pushpin', root).offset()!.top });
      $('.tabs', root).tabs();
    });
```

Reply:
No. Tried that before. That doesn't fix the problem.

Reply:
```suggestion
    this.$nextTick(() => {
      $('.scrollspy', root).scrollSpy();
      $('.pushpin', root).pushpin({ top: $('.pushpin', root).offset()!.top });

      M.Tabs.init($('.tabs', root));
    });
```

From what I can see the `setTimeout` ensures a [specific Materialize snippet](https://github.com/matomo-org/matomo/blob/6ceadb6ba2e7ff95baa91ecb18899f52180c49c5/plugins/CoreHome/javascripts/materialize-bc.js#L15) is executed before initializing the tabs. Using `this.$nextTick` seems to run before that code and then jQueryUI is used for the tabs :disappointed: